egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -225,6 +225,7 @@ def train_one_iteration(dir, iter, srand, egs_dir,
                         frame_subsampling_factor, truncate_deriv_weights,
                         run_opts,
                         dropout_edit_string="",
+                        use_multitask_egs=False,
```

Comment:
This function seems to be ignoring the 'use_multitask_egs' argument.  Was there a plan to add support for training chain models with multi-task egs later on?  I assume this would require code changes.